ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml

```
@@ -50,6 +50,10 @@ ref:
     documentation: |-
       This value represents the the DNS zone that is configured in the azure portal, and it will be used in the base_domain field
       of the install-config.yaml file.
+  - name: BASE_RESOURCE_GROUP
+    default: os4-common
+    documentation: |-
+      The base resource group where base domain resides, it is required when REGISTER_MIRROR_REGISTRY_DNS is enabled.
```

I didn't see `REGISTER_MIRROR_REGISTRY_DNS` is used in this step, in order to avoid misunderstandings, I'd suggest:
```suggestion
      The base resource group where base domain resides.
```